ovos_backend_client/backends/ovos.py

```diff
@@ -0,0 +1,222 @@
+        optional_params = optional_params or {}
+        if not lat_lon:
+            lat_lon = self._get_lat_lon(**optional_params)
+        params = {'input': query,
+                  "geolocation": "{},{}".format(*lat_lon),
+                  'units': units,
+                  **optional_params}
+        url = f'{self.url}/wolframalpha/simple/{self.uuid}'
+        data = self.get(url=url, params=params)
+        # TODO - this is returning an image ???
+        return data.text
+
+    def wolfram_full_results(self, query, units="metric", lat_lon=None, optional_params=None):
+        """Wrapper for the WolframAlpha Full Results v2 API.
+        https://products.wolframalpha.com/api/documentation/
+        Pods of interest
+        - Input interpretation - Wolfram's determination of what is being asked about.
+        - Name - primary name of
+        """
+        optional_params = optional_params or {}
+        if not lat_lon:
+            lat_lon = self._get_lat_lon(**optional_params)
+        params = {'input': query,
+                  "units": units,
+                  "mode": "Default",
+                  "format": "image,plaintext",
+                  "geolocation": "{},{}".format(*lat_lon),
+                  "output": "json",
+                  **optional_params}
+        url = f'{self.url}/wolframalpha/full/{self.uuid}'
+        data = self.get(url=url, params=params)
+        return data.json()
+
+    # Geolocation Api
+    def geolocation_get(self, location):
+        """Call the geolocation endpoint.
+
+        Args:
+            location (str): the location to lookup (e.g. Kansas City Missouri)
+
+        Returns:
+            str: JSON structure with lookup results
+        """
+        url = f"{self.url}/geolocate/location/config"
+        location = self.get(url, params={"address": location})
+        location = location.json()
+        if "timezone" not in location:
+            location["timezone"] = self._get_timezone(
+                lon=location["coordinate"]["longitude"],
+                lat=location["coordinate"]["latitude"])
+        return location
+
+    # STT Api
+    def stt_get(self, audio, language="en-us", limit=1):
+        """ Web API wrapper for performing Speech to Text (STT)
+
+        Args:
+            audio (bytes): The recorded audio, as in a FLAC file
+            language (str): A BCP-47 language code, e.g. "en-US"
+            limit (int): Maximum alternate transcriptions
+
+       """
+        content_type = "audio/x-flac"
+
+        data = self.post(OVOS_STT_URL, data=audio,
+                         params={"lang": language, "limit": limit},
+                         headers={"Content-Type": content_type})
+        if data.status_code == 200:
+            return [data.text]
+        raise RuntimeError(f"STT api failed, status_code {data.status_code}")
+
+    # Email API
+    def email_send(self, title, body, sender):
+        raise NotImplementedError()
```

Comment:
@AIIX can this be implemented in ovos api service? you can use the new smtp methods in ovos_utils package